server/src/main/java/io/deephaven/server/object/ObjectServiceGrpcImpl.java

```diff
@@ -262,6 +272,7 @@ public void fetchObject(
                     ObjectType objectTypeInstance = getObjectTypeInstance(type, o);
 
                     AtomicReference<FetchObjectResponse> singleResponse = new AtomicReference<>();
+                    AtomicBoolean isClosed = new AtomicBoolean(false);
```

Comment:
There's only one thread involved here, right? Would it make more sense for this to be `MutableObject` and `MutableBoolean`? Or do we think it's possible that the callee might do stuff asynchronously? If so, we're assuming they will block us while they wait for that to happen, it seems.

Reply:
`objectTypeInstance.clientConnection` could technically start something off thread

Reply:
It is technically possible that they go off thread and block - I'm not sure how much care we want to take with performance or correctness... I leaned towards correctness. I think this is actually more of a concern for a new plugin (async messaging tends to suggest other threads to get work done?), but somehow being loaded the old way
(if that is even possible?).

Either way, this will go away in a few months when we decide it is safe to remove fetchObject entirely.